api/filter_outputs.go

```diff
@@ -9,6 +9,36 @@ import (
 	"github.com/pkg/errors"
 )
 
+func (api *API) getFilterOutput(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	fID := chi.URLParam(r, "filter-output-id")
+
+	var filterOutput *model.FilterOutput
+
+	filterOutput, err := api.store.GetFilterOutput(ctx, fID)
+	if err != nil {
+		api.respond.Error(
+			ctx,
+			w,
+			statusCode(err),
+			Error{
+				err:     errors.Wrap(err, "failed to get filter output"),
+				message: "failed to get filter output",
+				logData: log.Data{
+					"id": fID,
+				},
+			},
+		)
+		return
+	}
+
+	resp := getFilterOutputResponse{
```

Comment:
Can we directly use *filterOutput in the api.respond.JSON without the need of this temp variable